src/collectors/systemd-journal.plugin/README.md

```diff
@@ -1,472 +1,629 @@
+- Error states and warnings have clear visual differentiation
 
-The histogram presented by the plugin is interactive:
+### UI navigation
 
-- **Zoom**, either with the global date-time picker or the zoom tool in the histogram's toolbox.
-- **Pan**, either with global date-time picker or by dragging with the mouse the chart to the left or the right.
-- **Click**, to quickly jump to the highlighted point in time in the log entries.
+The interface offers several ways to navigate logs:
+- Scroll through paginated results
+- Jump to specific timeframes
+- Click on histogram bars to focus on specific values
+- Use filter panels to narrow down results
+- Toggle between data views
 
-![image](https://github.com/netdata/netdata/assets/2662304/d3dcb1d1-daf4-49cf-9663-91b5b3099c2d)
+---
 
 ## PLAY mode
 
-The plugin supports PLAY mode to continuously update the screen with new log entries found in the journal files.
-Hit the "play" button at the top of the Netdata dashboard screen.
+The plugin supports PLAY mode for real-time log streaming. Click the ▶️ button at the top of the dashboard to activate it.
 
-On centralized log servers, PLAY mode provides a unified view of all the new logs encountered across the entire
-infrastructure,
-from all hosts sending logs to the central logs server via `systemd-remote`.
+- Continuously updates the screen with newly received logs
+- Works for both single nodes and centralized log servers
 
-## Full-text search
+:::tip
 
-The plugin supports searching through all fields of log entries using text patterns.
+**PLAY** mode offers a similar experience to `journalctl -f`, but with visual enhancements.
 
-**Pattern Matching**:
+:::
 
-- All patterns use "contains" matching (not exact match).
-- Patterns support wildcards (`*`) to match any characters.
-- Example: `error` matches "error", "errors", "error_count", etc.
-- Example with wildcard: `a*b` matches anything containing `a` followed by `b`, like `acb`, `a_long_b`.
+---
 
-**Multiple Patterns**:
+## Full text search
 
-- Separate patterns with `|` (pipe character).
-- Multiple patterns are combined with OR logic.
-- Example: `error|warning` matches entries containing either `error` OR `warning`.
+The plugin supports full-text search using flexible pattern matching:
 
-**Negative Patterns**:
+| Feature | Description |
+|---------|-------------|
+| Contains match | Default pattern style (e.g., `error` matches `error`, `error_count`) |
+| Wildcards | `*` matches any characters (e.g., `a*b` matches `acb`, `a_long_b`) |
+| Multiple patterns | Separate with `|` (e.g., `error|warning`) |
+| Negation | Prefix with `!` to exclude (e.g., `!systemd|*` excludes lines with `systemd`) |
 
-- Prefix pattern with `!` to exclude matching entries.
-- Example: `!systemd|*` matches any entry that does NOT contain `systemd`.
+:::tip
 
-## Query performance
+Full-text search applies across all fields. Combine with filters for precise results.
+
+:::
+
+---
+
+## Example commands
```

Comment:
This whole section seems strange to me, especially some of its subsections. Let's discuss offline.